front/components/assistant_builder/TeamSharingSection.tsx

```diff
@@ -0,0 +1,214 @@
+            assistant available to your team in the{" "}
+            <span className="font-bold">Assistant Gallery</span>. <br /> Your
+            team will be allowed to{" "}
+            <span className="font-bold">use and modify</span> the assistant.
+          </div>
+        </div>
+      </div>
+    );
+
+  if (initialScope === "private" && newScope === "published") {
+    return (
+      <div className="flex flex-col gap-3">
+        <div className="text-lg font-bold text-element-900">Team sharing</div>
+        <div className="flex items-center gap-3">
+          <div>
+            <Icon
+              visual={CheckCircleIcon}
+              size="lg"
+              className="text-success-500"
+            />
+          </div>
+          <div className="flex-grow text-sm text-element-700">
+            Assistant will be made available to your team in the{" "}
+            <span className="font-bold">Assistant Gallery</span>. <br />
+            Your team will be allowed to{" "}
+            <span className="font-bold">use and modify</span> the assistant.
+          </div>
+          <div>
+            <Button
+              variant="tertiary"
+              size="sm"
+              label="Cancel publication"
+              onClick={() => setNewScope("private")}
+            />
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  if (initialScope === "published" && newScope === "published") {
+    return (
+      <div className="flex flex-col gap-3">
+        <div className="text-lg font-bold text-element-900">Team sharing</div>
+        <div className="flex items-center gap-3">
+          <div className="flex-grow text-sm text-element-700">
+            This assistant can be discovered by everyone in the{" "}
+            <span className="font-bold">Assistant Gallery</span>. Users can add
+            it to their list to mention it in conversations. <br />
+            All members are allowed to{" "}
+            <span className="font-bold">modify and delete</span> the assistant.
+            You can create a duplicate to tweak your own, private version
+          </div>
+          <div>
+            <Tooltip label="Coming soon: you can create a duplicate to tweak your own, private version">
+              <Button
+                variant="tertiary"
+                size="sm"
+                label="Create duplicate"
+                onClick={() => false}
+                disabled={true}
+              />
+            </Tooltip>
+          </div>
+          {(owner.role === "admin" || owner.role === "builder") && (
+            <Tooltip label="Admins and builders can give an assitant workspace scope. It will then be only editable by admins and builders, and included by default in every member's list">
+              <Button
+                variant="tertiary"
+                size="sm"
+                label="Give workspace scope"
+                onClick={() => setNewScope("workspace")}
```

Comment:
The button + code is incorrect here. Fine to have Create a Duplicate disabled for now pending availability of duplication